tests/Api/Responses/admin/customer/get_customer_statistics_response.json

```
@@ -9,10 +9,11 @@
     "@id": "\/api\/v2\/.well-known\/genid\/@string@",
```

I believe, it show consist the following path:
```suggestion
    "@id": "\/api\/v2\/admin\/customers\/@integer@\/statistics",
```
The issue with the improper URI generation may be due to missing URI variable declarations here:
https://github.com/Sylius/Sylius/pull/16619/files#diff-3a862a839298d3078160ae01f1b7ed1c9e7be2529a5e2f6459ca18193be95755R66, but I'm not entirely certain.